pkg/util/util.go

```diff
@@ -50,7 +50,8 @@ const (
 	snapshotTotalElements = 6
 
 	// number of elements in backup Volume sources e.g. projects/{project name}/locations/{zone}/instances/{name}
-	volumeTotalElements = 6
+	singleShareVolumeTotalElements = 6
+	multiShareVolumeTotalElements  = 8
```

Comment:
can you add comment here to give an example of the string

Reply:
Added.